flood_api/routers/flood.py

```diff
@@ -1,21 +1,125 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, HTTPException
+from fastapi.responses import JSONResponse
 
-from flood_api.dependencies.flooddata import DetailedDataDep
-from flood_api.dependencies.queryparams import CoordinatesDep, DateRangeDep
+from flood_api.dependencies.flooddata import (DetailedDataDep,
+                                              SummaryDataDep,
+                                              ThresholdDataDep)
+from flood_api.dependencies.queryparams import (CoordinatesDep, 
+                                                DateRangeDep,
+                                                IncludeNeighborsDep)
+from flood_api.utils.geospatial_operations import get_data_for_point
+from flood_api.utils.json_utilities import (dataframe_to_geojson, 
+                                            custom_date_handler)
+from flood_api.utils.validation_helpers import point_within_roi
 
 router = APIRouter(tags=["flood"])
 
+@router.get("/summary", response_class=JSONResponse)
```

Comment:
Instead of JSONResponse, we should create a Pydantic Model that is used as the response.
The model should be well documented, so users of the API can get a full understanding of the expected response from this api-call.